iotdb/src/main/java/org/apache/iotdb/db/engine/overflow/io/OverflowResource.java

```diff
@@ -98,8 +99,9 @@ public OverflowResource(String parentPath, String dataPath, VersionController ve
       // the tail is at least the len of magic string
       insertIO = new OverflowIO(readWriter);
       readMetadata();
+    } catch (FileNotFoundException e){
+      LOGGER.debug("Failed to construct the OverflowIO.", e);
```

Comment:
Is "file not found" fine for all scenarios? (you set it as debug, so that by default, the log will be ignored)

Reply:
Opening the `insert overflow file` may throw the `FileNotFoundException`.
The error cased by the deletion of the file will be handled by the following `ioexception`
@jixuan1989